include/hal/library/requester/measlib.h

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
This header is (currently) not needed. In particular libspdm does not validate a large portion of the `MEASUREMENTS` response and instead leaves that up to the Verifier. If libspdm starts doing that then we'd start with `MEASUREMENTS` first.

Reply:
Thanks.  I have deleted the header.